airflow/ti_deps/deps/pool_slots_available_dep.py

```diff
@@ -62,9 +62,9 @@ def _get_dep_statuses(self, ti, session, dep_context=None):
             open_slots = pools[0].open_slots()
 
         if ti.state in STATES_TO_COUNT_AS_RUNNING:
-            open_slots += 1
+            open_slots += ti.pool_capacity
 
-        if open_slots <= 0:
+        if open_slots <= (ti.pool_capacity - 1):
             yield self._failing_status(
                 reason=("Not scheduling since there are %s open slots in pool %s",
```

Comment:
Extend this message to say how many slots we are looking for.

Reply:
modified the message to 
```
reason=("Not scheduling since there are %s open slots in pool %s "
                        "and require %s pool slots",
                        open_slots, pool_name, ti.pool_slots)
```